client/src/components/PortfolioFilters.tsx

```diff
@@ -234,6 +256,60 @@ export const PortfolioFilters = React.memo(
   })
 );
 
+const FiltersWrapper = (props: {
+  isMobile: boolean;
+  activeFilters: number;
+  resultsCount?: number;
+  children: React.ReactNode;
+}) => {
+  const { isMobile, activeFilters, resultsCount, children } = props;
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  return !isMobile ? (
+    <div className="filters">{children}</div>
+  ) : (
+    <FocusTrap
+      active={isOpen}
+      focusTrapOptions={{
+        clickOutsideDeactivates: true,
+        returnFocusOnDeactivate: false,
+        onDeactivate: () => setIsOpen(false),
+      }}
+    >
+      <div className="filters">
+        <details
+          className={classnames("filter filter-accordion filters-mobile-wrapper", {
+            active: !!activeFilters,
+          })}
+          open={isOpen}
+        >
+          <summary
+            onClick={(e) => {
+              e.preventDefault();
+              setIsOpen(!isOpen);
+            }}
+          >
+            <Trans>Filters</Trans>
+            {!isOpen && !!activeFilters && (
+              <span className="active-filter-count">{activeFilters}</span>
+            )}
+            {isOpen ? <CloseIcon className="closeIcon" /> : <ChevronIcon className="chevronIcon" />}
+          </summary>
+          <div className="dropdown-container scroll-gradient mobile-wrapper-dropdown">
+            {children}
+            {!!activeFilters && (
+              <button onClick={() => setIsOpen(!isOpen)} className="button is-primary">
+                <Trans>View Results</Trans>
+                {resultsCount && <span className="view-results-count">{resultsCount}</span>}
```

Comment:
this will evalutate to false when resultsCount is zero, do we want to show the "0" ?

Reply:
oh right, thanks for catching!